constants_local.xml

```diff
@@ -188,6 +192,8 @@
             <pubkey/>
         </peer>
     </lookups>
+    <upper_seed/>
+    <lower_seed/>
```

Comment:
Supposedly, the naming rule inside constant file should be \<sectionName\>\<TAG1\>...\</TAG1\>\<TAG2/\>...\</sectionName\>, is it possible to make it consistent?